site/docs/pages/wallet/types.mdx

```diff
@@ -82,6 +82,21 @@ export type WalletDropdownDisconnectReact = {
 };
 ```
 
+## `WalletDropdownFundLinkReact`
+
+```ts
+export type WalletDropdownFundLinkReact = {
+  className?: string; // Optional className override for the element
+  href: string;
+  icon?: ReactNode;
+  rel?: string;
+  openIn?: 'popup' | 'tab'; // Whether to open the funding flow in a tab or a popup window
+  target?: string; // Where to open the target if `openIn` is set to tab
```

Comment:
Thanks I was confusing the two

Reply:
I think overall there is an interesting question to ask. What are properties we want to be able to customize and what we don't want to have customized.

Otherwise it feels a weird overalap with the Link component.